joint_trajectory_controller/include/joint_trajectory_controller/trajectory.hpp

```diff
@@ -94,13 +96,16 @@ class Trajectory
    *      description above.
    * \param[out] end_segment_itr Iterator to the end segment for given \p sample_time. See
    *      description above.
+   * \param[in] progress_search_start_index If set to true, the next sample call will start
+   *      searching in the trajectory at the index of this call's result.
```

Comment:
This name makes me think the variable is an index. What do you think about renaming this to something like `search_monotonically_increasing`?

Reply:
```suggestion
   * \param[in] search_monotonically_increasing If set to true, the next sample call will start
   *      searching in the trajectory at the index of this call's result.
```